CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ pull request if there was one.
 
 ### Added:
 
+- [Move off BaseCompositePhysicalTable inheritance usage](https://github.com/yahoo/fili/issues/747)
```

Comment:
I don't remember, but I thought the PR would be linked here and the issue would be linked in the PR.